_includes/footer.html

```diff
@@ -1,7 +1,9 @@
+
 <footer>
     <div class="bml">
         <section class="container">
-            <p class="text-center"> Copyright &copy; 2019 IUS</p>
+            <p class="text-center"> Copyright &copy; 2020 IUS</p>
+            <h6 class="footer-text">The CSG will sponsor a student for class particpation if the student attendeds 2 lectures, participates in a competition, or becomes an officer.</h6>
```

Comment:
I think that should go somewhere other than the footer. Maybe another page?

Reply:
moved to its own section